src/data-types/schemas.ts

```diff
@@ -0,0 +1,102 @@
+export const TxMAIngress = {
```

Comment:
could we include the manual fraud interventions and the user led actions txma events as well - thanks